source/css/sass/mixins/_typography.scss

```diff
@@ -109,3 +110,12 @@
 @mixin label-tag-typography() {
   @include _label-typography($color-primary-normal);
 }
+
+@mixin covert-link($hover-color: $color-primary-dark) {
+  @include inherit-all($ensure: color text-decoration);
+  cursor: pointer;
```

Comment:
Tried to find a way to exclude certain properties inside `inherit-all()`. Closest I came was `cursor: revert`, but this doesn't work in many places, and would actually ignore any existing styles anyway. So, gave up and just set it back here. 🤷‍♂️